src/bionics_ui.cpp

```diff
@@ -50,14 +50,14 @@ char get_free_invlet( player &p )
     return ' ';
 }
 
-void show_bionics_titlebar( WINDOW *window, player *p, bionic_menu_mode mode )
+void draw_bionics_titlebar( WINDOW *window, player *p, bionic_menu_mode mode )
 {
     werase( window );
 
-    std::string pwr = string_format( _( "Power: %i/%i" ), int( p->power_level ),
-                                     int( p->max_power_level ) );
-    int desc_length = getmaxx( window ) - utf8_width( pwr ) - 1;
-    mvwprintz( window, 0, desc_length, c_white, "%s", pwr.c_str() );
+    const std::string pwr = string_format( _( "Power: %i/%i" ), int( p->power_level ),
+                                           int( p->max_power_level ) );
+    const int pwr_str_pos = getmaxx( window ) - utf8_width( pwr ) - 1;
```

Comment:
It would be nice to have a `right_print()` function (by analogy with the `center_print()`). This sort of right justification code is being used quite heavily. Also, it would make sense to rename the `mvwprintz()` (barely pronounceable) to `left_print()` and mercilessly kill its twin sister, the `printz()`. It's unused anyway.

These suggestions are BTW (apart from the PR).